core/src/test/java/org/hibernate/ogm/backendtck/dialectinvocations/GridDialectOperationInvocationForOneToOneTest.java

```diff
@@ -14,6 +14,8 @@
 
 import org.hibernate.Session;
 import org.hibernate.Transaction;
+import org.hibernate.ogm.backendtck.associations.onetoone.Husband;
```

Comment:
I would prefer having separate as well, but we usually don't change existing models so it does not bother me too much.